### PR TITLE
3.113.0

### PR DESCRIPTION
**Pull request by Razz4780**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,76 @@ This project uses [*towncrier*](https://towncrier.readthedocs.io/) and the chang
...
+
+  ```
+  mirrord container [options] -- <docker/podman> run ...
+  ```
+
+  Because we need to run internal proxy process on the same network as the
+  process loaded with `mirrord-layer`, to keep config and kubernetes
+  comparability the communication to mirrord agent is made via external proxy
+  that will run on the host machine.
+  ```
```
Comment (DmitryDodzin) [inline CHANGELOG.md:26]: I just have a dilemma, I like the graphic but. not sure should be in changelog

Comment (Razz4780) [reply CHANGELOG.md:26]: Nah, I think the graphic is cool. Why not?

Comment (DmitryDodzin) [reply CHANGELOG.md:26]: It's just Iv'e never seen one in any changelog

Comment (Razz4780) [reply CHANGELOG.md:26]: @eyalb181 wdyt?

Comment (eyalb181) [reply CHANGELOG.md:26]: Let's keep it in, it's attention grabbing

**Pull request merged**